samples/springmvc-sample/springmvc-provider/src/main/java/io/servicecomb/samples/springmvc/provider/SpringmvcHelloImpl.java

```diff
@@ -73,7 +74,7 @@ public String metricsForTest() {
     try {
```

Comment:
why not return Map<String, String>

each time you write JsonUtils.writeValueAsString to be a schema operation result, you need think if you have make something wrong.

Reply:
Done,we don't need this code and forget delete;(IT test code had remove to demo-springmvc)